R/pev_parameters.R

```diff
@@ -86,12 +85,13 @@ set_pev_epi <- function(
   timesteps,
   age,
   min_wait,
-  booster_timestep,
+  booster_spacing,
```

Comment:
Should we have a check that the vector of booster spacings are monotonically increasing when more than one booster is used - setting otherwise gives quite a cryptic `Error: delay must be >= 0` error